spanner/changestreams/src/main/java/com/example/spanner/changestreams/QueryChangeStreamSample.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner.changestreams;
+
+import com.example.spanner.changestreams.model.ChangeStreamRecord;
+import com.example.spanner.changestreams.model.ChildPartition;
+import com.example.spanner.changestreams.model.ChildPartitionsRecord;
+import com.example.spanner.changestreams.model.DataChangeRecord;
+import com.example.spanner.changestreams.model.HeartbeatRecord;
+import com.google.cloud.Timestamp;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.Statement;
+import com.google.common.collect.ImmutableList;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Sample code for querying change stream, it:
+ * 1. Insert test data into the table.
+ * 2. Execute a change stream initial query to get change stream partition tokens.
+ * 3. Execute a change stream partition query to get data change records of the inserted rows.
+ */
+public class QueryChangeStreamSample {
+  public static void main(String... args) {
+    if (args.length != 3) {
+      Utils.printUsage(QueryChangeStreamSample.class.getName());
+      return;
+    }
+
+    final String instanceId = args[0];
+    final String databaseId = args[1];
+    final String prefix = args[2];
+    final String tableName = prefix + "_Singers";
+    final String changeStreamName = prefix + "_ChangeStreamSingers";
+
+    SpannerOptions options = SpannerOptions.newBuilder().build();
```

Comment:
We should not build the options multiple times (in this class and in the Utils class). I think we should concentrate the building and getting the service in one class, which performs all the operations we would like.

Reply:
Done.